demo/scripts/controls/contextMenu/ResetListPlugin.ts

```diff
@@ -42,6 +46,75 @@ export default class ResetListPlugin implements ContextMenuProvider<ContextMenuI
                     }
                 );
             }
+            if (
+                safeInstanceOf(list, 'HTMLOListElement') ||
+                safeInstanceOf(list, 'HTMLUListElement')
+            ) {
+                items.push({
+                    key: 'adjustListIndents',
+                    name: 'Adjust List Indents',
+                    dropDownItems: [
+                        {
+                            key: 'numberPosition',
+                            name: 'Number Position',
+                            onClick: () => {
+                                let value = parseInt(prompt('Set Value to...', '1'));
+                                let unitValue = prompt(
+                                    "Set Value to... ('em' | 'px' | '%' | 'in')",
+                                    'em'
+                                );
+
+                                switch (unitValue) {
+                                    case 'em':
+                                        setListNumberPosition(this.editor, list, value, 'em');
+                                        break;
+                                    case 'px':
+                                        setListNumberPosition(this.editor, list, value, 'px');
+                                        break;
+                                    case '%':
+                                        setListNumberPosition(this.editor, list, value, '%');
+                                        break;
+                                    case 'in':
+                                        setListNumberPosition(this.editor, list, value, 'in');
+                                        break;
+                                    default:
```

Comment:
You can merge them into:
case 'em':
case 'px':
case '%':
case 'in':
  setListNumberPosition(this.editor,list,value, unitValue);
  break;